extra/1-radio-stations.js

```diff
@@ -12,6 +12,13 @@
  * - Create an array  starting at 87 and ending in 108
  * - Should return this array to use in other functions
  */
+ let freqArray=[];
+function getAllFrequencies(){
+  for(let i=0; i<22; i++){
+    freqArray[i]= i+87;
+  }
+  return freqArray;
+}
```

Comment:
There is a simpler way of doing this

```
for (let i=87; i<=108; i++){
  freqArray.push(i);
}
```
This will be much clearer to a developer reading this code in the future, who might not understand what 22 and 87 mean.